components/builder-web/app/origin/origin-page/origin-integrations-tab/_origin-integrations-tab.component.scss

```diff
@@ -1,5 +1,5 @@
 .integrations-tab {
-  @include span-columns(8);
+  @include span-columns(10);
```

Comment:
It looks from the mock like these should be 9 and 3 rather than 10 and 2.

Reply:
I made it 10/2 because the text of the boxes wouldn't fit in a 9/3

Reply:
Do you mean the button labels? When I change to 9/3, things look all right:

![image](https://user-images.githubusercontent.com/274700/32066820-e38ab206-ba35-11e7-91c5-89ab84748cff.png)